Compilation warnings on Linux/gcc 10.1

```
warning: :nifs, :cnodes and :ports keys are deprecated. Use :natives instead
  (bundlex 0.3.0) lib/bundlex/project.ex:159: Bundlex.Project.convert_input_config/1
  (bundlex 0.3.0) lib/bundlex/project.ex:122: Bundlex.Project.get/1
  (bundlex 0.3.0) lib/tasks/compile.bundlex.ex:27: Mix.Tasks.Compile.Bundlex.run/1
  (mix 1.10.3) lib/mix/task.ex:330: Mix.Task.run_task/3
  (mix 1.10.3) lib/mix/tasks/compile.all.ex:76: Mix.Tasks.Compile.All.run_compiler/2

==> example
Bundlex: Building natives: example, example, example, example_cnode, example_nif, example_port
/home/bblaszkow/swmansion/membrane/bundlex/test_projects/example/c_src/example/foo_nif.c: In function ‘export_foo’:
/home/bblaszkow/swmansion/membrane/bundlex/test_projects/example/c_src/example/foo_nif.c:4:52: warning: unused parameter ‘argc’ [-Wunused-parameter]
    4 | static ERL_NIF_TERM export_foo(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
      |                                                ~~~~^~~~
/home/bblaszkow/swmansion/membrane/bundlex/test_projects/example/c_src/example/bar_nif.c: In function ‘export_bar’:
/home/bblaszkow/swmansion/membrane/bundlex/test_projects/example/c_src/example/bar_nif.c:4:52: warning: unused parameter ‘argc’ [-Wunused-parameter]
    4 | static ERL_NIF_TERM export_bar(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
      |                                                ~~~~^~~~
Generated example app
```

--- a/test_projects/example/c_src/example/bar_nif.c
+++ b/test_projects/example/c_src/example/bar_nif.c
@@ -3,6 +3,7 @@
 
 static ERL_NIF_TERM export_bar(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
 {
+    (void)argc;
     int a, b;
     enif_get_int(env, argv[0], &a);
     enif_get_int(env, argv[0], &b);

--- a/test_projects/example/c_src/example/foo_nif.c
+++ b/test_projects/example/c_src/example/foo_nif.c
@@ -3,6 +3,7 @@
 
 static ERL_NIF_TERM export_foo(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
 {
+    (void)argc;
     int a, b;
     enif_get_int(env, argv[0], &a);
     enif_get_int(env, argv[0], &b);

